.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Production build
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Use Node.js ${{ matrix.node-version }}'
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ matrix.node-version }}'
+
+      - run: npm i
+      - run: npm run build
+      - uses: papeloto/action-zip@v1
+        with:
+          files: dist/
+          recursive: true
+          dest: monitor.zip
+
+      - uses: "marvinpinto/action-automatic-releases@latest"
```

Comment:
this repo seems kinda small for something that is super common... perhaps there is a reason for that?  
May be a good idea to try to find another one.